### ISSUE TITLE
where:stop doesn't work

### ISSUE THREAD

**Issue by GoogleCodeExporter** [baluragala/gtalksms#359]
```
What steps will reproduce the problem?
1. send where
2. wait a few messages to arrive
3. send where:stop
4. wait for the "Stop locating phone" message

What is the expected output?
No more location messages are received

What do you see instead?
More location messages are received.

Additional informations:
LG-G2, CyanogenMod v11 M6

```

Original issue reported on code.google.com by `vincenzo...@notorand.it` on 16 May 2014 at 9:28

**Comment by GoogleCodeExporter** [comment]
Original comment by `Florent....@gmail.com` on 17 Aug 2014 at 2:37
- Changed state: **FixedInBeta**

**Comment by GoogleCodeExporter** [comment]
```
Released on Play Market as 5.0
```

Original comment by `Florent....@gmail.com` on 25 Oct 2014 at 6:23
- Changed state: **Deployed**
